content/en/tools/digital-currencies/_index.md

```diff
@@ -0,0 +1,24 @@
+---
+date: '2023-03-21'
+section_menu: tools
+menu:
+  tools:
+    name: Digital Currencies
+    weight: 600
+    identifier: digital-currencies
+title: Welcome to Digital Currencies
+description: "Documentation describing the Digital Currencies feature from R3"
```

Comment:
who is our audience here? what is the purpose of this document ?

Reply:
I've added the following:

"Welcome to the documentation for the Digital Currencies for Corda 4 GUI. The purpose of this documentation is to describe the concepts behind Digital Currencies, show the lifecycle of a digital token definition, describe the functionality available to participants in a digital currency network, and how these action can be performed via the Digital Currencies GUI."